src/backend/distributed/commands/multi_copy.c

```diff
@@ -1075,23 +1153,26 @@ SendCopyBinaryFooters(CopyOutState copyOutState, List *connectionList)
  * shard.
  */
 static StringInfo
-ConstructCopyStatement(CopyStmt *copyStatement, int64 shardId)
+ConstructCopyStatement(CopyStmt *copyStatement, int64 shardId, bool useBinaryCopyFormat)
 {
 	StringInfo command = makeStringInfo();
 
-	char *schemaName = copyStatement->relation->schemaname;
-	char *relationName = copyStatement->relation->relname;
-
-	char *shardName = pstrdup(relationName);
+	char *shardSchemaName = copyStatement->relation->schemaname;
+	char *shardTableName = pstrdup(copyStatement->relation->relname);
```

Comment:
For the future, please don't perform independent refactoring in the same commit as an independent functional change.